wbstack/src/Settings/LocalSettings.php

```diff
@@ -218,6 +218,18 @@
 $wgPasswordSender = 'noreply@' . getenv('MW_EMAIL_DOMAIN');
 $wgNoReplyAddress = 'noreply@' . getenv('MW_EMAIL_DOMAIN');
 
+// SMTP
+if (getenv('MW_MAILGUN_ENABLED') !== 'yes') {     // do not configure SMTP if Mailgun Extension is wanted
+    $wgSMTP = [
+        'host'     => getenv('MW_SMTP_HOST'),     // could also be an IP address. Where the SMTP server is located. If using SSL or TLS, add the prefix "ssl://" or "tls://".
+        'IDHost'   => getenv('MW_EMAIL_DOMAIN'),  // Generally this will be the domain name of your website (aka mywiki.org)
+        'port'     => getenv('MW_SMTP_PORT'),     // Port to use when connecting to the SMTP server
+        'auth'     => getenv('MW_SMTP_AUTH'),     // Should we use SMTP authentication (true or false)
```

Comment:
Should this even be configurable or always set to true?

Reply:
I think true always would be a safe initial bet
Buts it is already configurable here, so we could leave it!